test/test-helpers/_setup.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs-extra');
+const path = require('path');
+
+const fixtures = path.join(process.cwd(), 'test', 'fixtures');
+
+fs.renameSync(path.join(fixtures, 'repo-with-tags', 'git'), path.join(fixtures, 'repo-with-tags', '.git'));
```

Comment:
try to rename it like following (I think, it is easier to maintain): https://github.com/JPeer264/node-git-commit-info/blob/master/test.js#L10-L21